### PR TITLE
James h65 patch 3#compliance

### PR DESCRIPTION
**Pull request by JamesH65**
Have now updated the compliance page to include all the docs available.

@Roger-Thornton @LornaLynch @ghollingworth @jimbojr for proof reading/acceptance testing!

### CODE REVIEW

File: hardware/raspberrypi/conformity.md [hunk 1]
```diff
@@ -1,6 +1,83 @@
...
+- USA
+  - ANSI C63.4 
+  - FCC 47 CFR Part 15, Subpart B, Class B Digital Device.
+
+ [FCC Declaration of Conformity](https://www.raspberrypi.org/files/compliance/Raspberry_Pi_2B_V1-2_DOC_FCC.pdf)   
+ [EU Declaration of Conformity](https://www.raspberrypi.org/files/compliance/Raspberry_Pi_2B_V1-2_DOC_CE.pdf)
+
+## Raspberry Pi Model 3
+
+The Raspberry Pi Model 3 devices has undergone extensive compliance testing and meet the following standards.
```
Comment (lurch) [inline hardware/raspberrypi/conformity.md]: Either "devices have" or "device has"

File: hardware/raspberrypi/conformity.md [hunk 2]
```diff
@@ -1,6 +1,83 @@
 # Compliance
 
-Raspberry Pi models conform to various standards. Those documents published are linked below.
+Raspberry Pi models conform to various standards. The available Documents of Conformance are linked below. Note that not all standards that apply are listed here.
+
+## Raspberry Pi Model A+
+
+The Raspberry Pi Model A+ device has undergone extensive compliance testing and meet the following standards.
```
Comment (lurch) [inline hardware/raspberrypi/conformity.md]: meet -> meets
(and same change also elsewhere)

File: hardware/raspberrypi/conformity.md [hunk 3]
```diff
@@ -17,3 +94,41 @@ All Raspberry Pi Compute Module 3 devices and the IO board have undergone extens
 
  [FCC Declaration of Conformity](https://www.raspberrypi.org/files/compliance/RaspberryPiCM3_DOC_FCC.pdf)   
  [EU Declaration of Conformity](https://www.raspberrypi.org/files/compliance/RaspberryPiCM3_DOC_EU.pdf)
+
+## Raspberry Pi Camera, PiNoIR camera, Pi Camera 2, PiNoIR Camera 2
+
+The Raspberry Pi cameras have undergone compliance testing and meets the following standards.
```
Comment (lurch) [inline hardware/raspberrypi/conformity.md]: meets -> meet

**Comment by lurch** [comment]
Is there no FCC conformity doc for the RPi3?

**Comment by JamesH65** [comment]
@lurch @Roger-Thornton  the collated set of documents I've seen did not have an FCC for RPi3, although it does have it.

**Comment by JamesH65** [comment]
I think this can be copy edited as it stands - we can always add extra documents as I incorporate them later.

**Pull request merged**